packages/youbet-sdk/package.json

```diff
@@ -6,7 +6,8 @@
   "module": "./dist/esm/index.js",
   "exports": {
     "import": "./dist/esm/main.js",
-    "require": "./dist/cjs/main.js"
+    "require": "./dist/cjs/main.js",
+    "types": "./dist/esm/main.d.ts"
```

Comment:
what's the difference for types line?

Reply:
`exports` is a more modern standard usage, but my editor indicated that the type could not be found. Adding it resolved the issue.